hw/mcu/dialog/include/DA1469xAB.h

```diff
@@ -8588,6 +8588,16 @@ typedef struct {                                /*!< (@ 0x50000100) WAKEUP Struc
 
 /** @} */ /* End of group PosMask_peripherals */
 
+/* Definitions for REMAP_ADR0 (bits 2:0) of SYS_CTRL_REG. See Datasheet for details */
```

Comment:
The header says:

```
 * @file     DA1469xAB.h
 * @brief    CMSIS HeaderFile
 * @version  1.2
 * @date     28. March 2019
 * @note     Generated by SVDConv V3.3.25 on Thursday, 28.03.2019 17:24:46
 *           from File 'DA1469x.xml',
```
I've no idea if this file is ever gonna be re-generated, in which case modifying it wouldn't be so great. Maybe it would be better to just define this inside the crypto driver itself.

Reply:
Moved the REMAP_ADR0 defines to crypto driver.